app/src/main/java/com/asfoundation/wallet/advertise/WalletPoAService.java

```diff
@@ -85,15 +84,16 @@
         // set the chain id received from the application. If not received, it is set as the main
         String packageName = intent.getStringExtra(PARAM_APP_PACKAGE_NAME);
         requirementsDisposable =
-            proofOfAttentionService.isWalletReady(intent.getIntExtra(PARAM_NETWORK_ID, 1))
+            proofOfAttentionService.isWalletReady(intent.getIntExtra(PARAM_NETWORK_ID, -1))
```

Comment:
The idea is to have the main network as the default network value when there isn't a value available in the intent extras.